components/displays/HD44780-i2c/README.md

```diff
@@ -0,0 +1,57 @@
+The following files allow using LCD displays based on HD44780 connected via i2c bus for this project. The following displays have been used for testing:
+- 2x16 display
+- 4x20 display (recommended as more information can be displayed)
+
+Various informations such as artist, album, track_number, track_title, track_time and many more can be displayed see main script for more display options.
+
+The required files are:
+- components/displays/HD44780-i2c/i2c_lcd.py
+- components/displays/HD44780-i2c/i2c_lcd_driver.py
+- components/displays/HD44780-i2c/i2c-lcd.service.default.sample
+- components/displays/HD44780-i2c/README.md
+
+
+The first file is the main LCD script that makes use of I2C_LCD_driver.py.
+The second file is the library needed to drive the LCD via i2c, originates from DenisFromHR (Denis Pleic) see http://www.circuitbasics.com/raspberry-pi-i2c-lcd-set-up-and-programming
+The third is used as sample service file that runs the i2c_lcd.py main script at boot-up if the service is properly installed (install description can be found below.).
+The fourth file is this file which describes the features, usage and installation of the code.
+
+### Installation
+
+1. You need to install additional python libraries. Run the following two command in the command line:
+`sudo apt-get install i2c-tools python-smbus python3-numpy python-mpdclient python-mpd`
+`pip install smbus numpy python-mpd2`

```

Comment:
Could you add a requirements.txt file with all necessary python packages?

Reply:
I will do so, soon.

Reply:
@SimonChelkowski, isn't this redundant to install the apt packages and one line later the same(?) pip packages?